packages/helpers/classes/mail.js

```diff
@@ -247,7 +273,7 @@ class Mail {
     ) {
       throw new Error('Provide at least one of to, cc or bcc');
     }
-    this.addPersonalization(new Personalization({to, cc, bcc}));
```

Comment:
The code style in this package doesn't use leading/trailing spaces after/before curly brackets, could you revert this change?